src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -445,8 +554,26 @@ public float Derivative(float output, float label)
     }
 
     /// <summary>
-    /// Tweedie loss, based on the log-likelihood of the Tweedie distribution.
+    /// Tweedie loss, based on the log-likelihood of the Tweedie distribution. This loss function is used in Tweedie regression.
     /// </summary>
+    /// <remarks type="text/markdown"><![CDATA[
+    ///
+    /// The Tweedie Loss function is defined as:
+    ///
+    /// \[
+    /// L(f(\vec{ x}, y, i) =
+    /// \begin{cases}
```

Comment:
@natke @sfilipi do you know if this will render? This is using the amsmath LaTeX package.